lib/suppressionList.js

```diff
@@ -64,7 +64,7 @@ module.exports = function(client) {
       // Check for bulk upsert
       if(recipient.constructor === Array) {
         options.uri = api;
-        options.json = toApiFormat(recipient);
+        options.json = { recipients: toApiFormat(recipient) };
       } else if(!recipient.email) {
```

Comment:
Single recipient upload isn't supported anymore, so this else block should just error out with the same error we were getting from the API. Alternatively if could insert the single value into an array.